src/components/CloseModal.js

```diff
@@ -53,8 +53,8 @@ CloseModal.propTypes = {
 };
 
 CloseModal.defaultProps = {
-  title: <FormattedMessage id="wizard.closeTitle" defaultMessage="Exit source creation?" />,
-  exitTitle: <FormattedMessage id="wizard.exitText" defaultMessage="Exit" />,
+  title: <FormattedMessage id="wizard.closeTitle" defaultMessage="Cancel creating the integration?" />,
+  exitTitle: <FormattedMessage id="wizard.exitText" defaultMessage="Cancel" />,
   stayTitle: <FormattedMessage id="wizard.stayText" defaultMessage="Stay" />,
```

Comment:
Per yesterday's discussion about similar things in RBAC, these should probably be "Discard" and "Cancel"
https://redhat-internal.slack.com/archives/C05M0SNTLM8/p1706801711430029

Reply:
Same as previous comment. This was directly suggested by UX team.